warehouse/bigquery.go

```diff
@@ -37,11 +37,35 @@ func NewBigQuery(c *config.Config) *BigQuery {
 	if c.GCS.GCSOnly {
 		log.Printf("Config flag GCSOnly is on, data will not be loaded to BigQuery")
 	}
+
 	return &BigQuery{
 		conf: c,
 	}
 }
 
+// GetExportTableColumns returns a slice of the columns in the existing export table
+func (bq *BigQuery) GetExportTableColumns() []string {
+	var columns []string
+	var err error
+	var md *bigquery.TableMetadata
```

Comment:
Any reason why `md` and `err` is predeclared? In go, I believe the usual pattern is to create variables on the fly.

Reply:
+1